mono_collection/mono_collection.postman_collection.json

```
@@ -0,0 +1,4778 @@
+{
+	"info": {
+		"_postman_id": "60e3c4dd-b172-4eb3-8356-d9135eb5e97f",
+		"name": "mono_collection",
```

mono collection is useful for us internally, but this will be used by customers externally. Below is only a suggestion on what to name the collection:

```suggestion
		"name": "OneLogin API Definitive",
```